shuup/core/models/_products.py

```diff
@@ -494,7 +494,11 @@ def get_price(self, context, quantity=1):
         :type context: shuup.core.pricing.PricingContextable
         :rtype: shuup.core.pricing.Price
         """
-        return self.get_price_info(context, quantity).price
+        # Check if product has variations (children) if yes, return cheapest variation price
+        if self.is_variation_parent():
+            return self.get_cheapest_child_price_info(context, quantity).price
+        else:
+            return self.get_price_info(context, quantity).price
 
     def get_base_price(self, context, quantity=1):
```

Comment:
How about `get_base_price` and `get_price_info`? Maybe this is something that the `shuup.core.pricing.get_price_info` should deal with. Also I don't like that we do this for front product sort which likely has some bigger issues in it.

Reply:
Maybe even something pricing module should deal with since that is likely only thing that is responsible how this kind of situation is handled. Maybe in some implementations or projects the cheapest price is not shown but price for some specific variation etc...